File: app/src/main/java/tw/com/louis383/coffeefinder/maps/MapsClickHandler.java
package tw.com.louis383.coffeefinder.maps;

import tw.com.louis383.coffeefinder.model.domain.CoffeeShop;

/**
 * Created by louis383 on 2017/3/7.
 */

public interface MapsClickHandler {
    void onMapClicked();
    void onMarkerClicked(CoffeeShop coffeeShop);
}

File: app/src/main/java/tw/com/louis383/coffeefinder/maps/MapsClickHandler.kt
package tw.com.louis383.coffeefinder.maps

import tw.com.louis383.coffeefinder.model.data.entity.Shop

/**
 * Created by louis383 on 2017/3/7.
 */

interface MapsClickHandler {
    fun onMapClicked()
    fun onMarkerClicked(coffeeShop: Shop)
}
